snapshot/git/gitdb/setup.go

```diff
@@ -57,3 +62,10 @@ func (i *NewRepoIniter) Init() (*repo.Repository, error) {
 
 	return repo.InitRepo(repoTmp.Dir)
 }
+
+// Noop Repo Updater
+type NewRepoUpdater struct{}
```

Comment:
updating this, should make more sense. Also removed ResourceUpdater in snapshot.DB and just reused Updater from snapshot/filer.